_posts/multiworld/2024-03-10-multiworld-with-nintendont.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "Connecting Randovania to a Real Wii or Wii U with Nintendont Multiworld"
+tags: multiworld prime1 prime2
+---
+# Connecting Randovania to a Real Wii or Wii U with Nintendont Multiworld #  
+While playing Metroid Prime or Metroid Prime 2: Echoes Multiworld Randomizer on Dolphin Emulator is the most popular method, you can also play it on a real Wii or Wii U (Wii Mode) with a [custom fork of Nintendont](https://github.com/randovania/Nintendont/) with support for networked multiworld and automatic item tracking.  
+Using a real Wii and Wii U console may be favorable if your computer does not run emulated GameCube games well.  
+  
+## Requirements ##  
+  
+* **A Computer with an SD Card Reader**  
+  This guide assumes using Windows, but this should be similar for other operating systems.  
+* **All the prerequisites to use and generate a game with [Randovania](https://randovania.org/)**  
+* **An SD Card**  
+  Recommended format: FAT32 with 32 KB clusters  
+  * **⚠️ WARNING** Formatting will erase all the data on that card. Backup/copy your card's files to a different location before formatting.  
+  * exFAT can work, but can cause issues with other homebrew apps. NTFS will not work.  
+  * USB Storage may also work, but it's difficult to tell which models are reliable with the Wii and Wii U.  
+* A **Wii** or **Wii U (Wii Mode)** console with:  
+  * **The Homebrew Channel**  
+    * [Installation Guide](https://wii.hacks.guide/get-started)  
+      (Patched/cIOSes are not needed.)  
+  * **Connection to the Network**  
+    * **ℹ️ NOTE** It is strongly recommended to use a wired Ethernet connection for the console instead of the console's built-in Wi-Fi if possible. The console's Wi-Fi is *slow*.  
```

Comment:
speed really isn't the issue here. Wi-Fi has two main issues: 1) stability. this instability is the cause of echoes hanging at boot, for instance 2) some routers have privacy settings that the Wii and vWii cannot navigate in the Homebrew Channel

Reply:
isnt there also some issue where the wii can only connect to certain protocols? last time i tried connecting my wii to my network it just didnt want to connect at all.

Reply:
that is the second problem I mentioned

Reply:
may have misunderstood, but for me it wasnt even an HBC thing. The wii just failed all connection tests in the wii system settings.